src/styles/blocks/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  padding: 50px $cell-padding 0;
+
+  width: ($square-size * 7) + ($cell-padding * 4);
+
+  @each $day, $margin in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $margin;
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} {
+      & :nth-child(1n + #{$i + 1}) {
```

it's not  necessary to write `1` in this formula
```suggestion
      & :nth-child(n + #{$i + 1}) {
```